db/schema.rb

```diff
@@ -70,6 +70,16 @@
     t.boolean "hide_recipients"
   end
 
+  create_table "emails", force: :cascade do |t|
+    t.boolean  "hide_recipients"
+    t.string   "recipients"
+    t.string   "reply_to"
+    t.string   "subject"
+    t.string   "content"
+    t.datetime "created_at",      null: false
+    t.datetime "updated_at",      null: false
+  end
+
```

Comment:
I am not sure why the block above have been added? Otherwise it looks very good 👍

Reply:
Huh. Good point.